### PR TITLE
Fixed typos in README.md

### PR DESCRIPTION
**Pull request by IsaacPayne**
Fixed a handful of typos in the ReadME

### CODE REVIEW

**Comment by danger-awesome-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Found 1 link issue</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Too many changes (when adding, please keep it to one project per Pull Request)</td>
    </tr>
  </tbody>
</table>

#### Link issue by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
2091 | [301](https://httpstatuses.com/301) | https://github.com/jerkoch/SwipeCellKit redirects to<br>https://github.com/SwipeCellKit/SwipeCellKit


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: README.md [hunk 1]
```diff
@@ -2449,7 +2449,7 @@ Most of these are paid services, some have free tiers.
 * [Refactorator](https://github.com/johnno1962/Refactorator) - Xcode Plugin that Refactors Swift & Objective-C :large_orange_diamond:
 * [VWInstantRun](https://github.com/wangshengjia/VWInstantRun) - An Xcode plugin let you build & run your selected lines of code in Xcode without running the whole project, you'll have the output instantly in your Xcode console. :large_orange_diamond:
 * [TTPasteHistory](https://github.com/tutumagi/TTPasteHistory) - A Xcode plugin. Recording you copy-and-paste history easily to write the code
-* [xSendIssue](https://github.com/hungri-yeti/xSendIssue) - Plugin for Xcode to submit github issues directly from within Xcode.
+* [xSendIssue](https://github.com/hungri-yeti/xSendIssue) - Plugin for Xcode to submit Github issues directly from within Xcode.
```
Comment (dkhamsing) [inline README.md]: this should say GitHub

**Comment by dkhamsing** [comment]
@IsaacPayne wow thank you!

File: README.md (hunk 1 above)
Comment (IsaacPayne) [inline README.md]: Fixed

**Comment by dkhamsing** [comment]
Thank you for contributing to `awesome-ios` 😄

**Comment by dkhamsing** [comment]
`GutHub` 👀

**Pull request merged**